spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/SubscriberFactory.java

```diff
@@ -16,15 +16,23 @@
 
 package org.springframework.cloud.gcp.pubsub.support;
 
+import com.google.api.gax.retrying.RetrySettings;
 import com.google.cloud.pubsub.v1.MessageReceiver;
 import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.cloud.pubsub.v1.stub.SubscriberStub;
+import com.google.pubsub.v1.PullRequest;
 
 /**
  * Generates {@link Subscriber} objects.
```

Comment:
This JavaDoc isn't actual any more. Moreover would be great to have JavaDocs on the methods as well.